db/migrate/20210628150336_create_failures.rb

```diff
@@ -0,0 +1,9 @@
+class CreateFailures < ActiveRecord::Migration[6.0]
+  def change
+    create_table :failures do |t|
+      t.jsonb :data
+      t.datetime :created_at, null: false
```

Comment:
```suggestion
      t.datetime :created_at, null: false, index: true
```
Le pondría un índice, porque seguro después en el listado los ordenamos por fecha.

Reply:
perfecto